### PR TITLE
Added skip conditions to blocks

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
Have added the skip condition to blocks
Have also added a not contains rule

### How to review 
Run through the new test schema 'test_skip_condition_block.json' and observe that the block is skipped.

### CODE REVIEW

**Review by benjefferies — COMMENTED** [review]
Are we going to follow up the frontend navigation on another card?

File: app/questionnaire/rules.py [hunk 1]
```diff
@@ -23,6 +23,8 @@ def evaluate_rule(when, answer_value):
         return True
     elif condition == 'contains' and isinstance(answer_to_test, list) and match_value in answer_to_test:
         return True
+    elif condition == 'not contains' and isinstance(answer_to_test, list) and match_value not in answer_to_test:
```
Comment (benjefferies) [inline app/questionnaire/rules.py:26]: We should have some tests for 'not contains' conditions

Comment (jonnyshaw89) [reply app/questionnaire/rules.py:26]: Like this one? https://github.com/ONSdigital/eq-survey-runner/pull/902/files#diff-c0704fcf066dc43ec2f73117f8ca828eR110

Comment (benjefferies) [reply app/questionnaire/rules.py:26]: err yeah

**Review by benjefferies — APPROVED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**